tests/e2e/specs/price-benchmark/price-benchmark-suggestions.test.js

```diff
@@ -222,6 +222,44 @@ test.describe( 'Price Benchmark Page', () => {
 				'There was an error loading the price benchmark suggestions.'
 			);
 		} );
+
+		test( 'FAQ link is present and clickable', async () => {
+			await priceBenchmarkPage.fulfillPriceBenchmarkSuggestions( [
+				...priceBenchmarkSuggestionsData,
+			] );
+			// Get the faq link by .dataviews__view-actions > .components-external-link by text.
+			const faqLink = page.locator(
+				'.dataviews__view-actions .components-external-link'
+			);
+			await expect( faqLink ).toBeVisible();
+			await expect( faqLink ).toContainText( 'FAQ' );
+		} );
+
+		test( 'Clicking the FAQ link opens the correct URL', async () => {
```

Comment:
@ankitrox I feel this is unnecessary. As long as we test if the link href attribute has the correct URL, that should be enough.